### ISSUE TITLE
#!s in spotipy-tests are platform dependent

### ISSUE THREAD

**Issue by manalili18** [manalili18/tyler-sandbox#2]
This is a minor issue/inconvenience. The path that I use for a spotipy-enabled python is platform dependent right now. (It's located at /Users/pmanalil/<something>/anaconda3/python) I think this is a side effect of the pip installer.

**Comment by tschmalenbach** [comment]
This could also be an issue with anaconda. the system tends to look in default python locations even when using the conda environment.

**Comment by manalili18** [comment]
Yes, that's exactly what happens. Do you know if there is a way to install python packages to /usr/bin/python instead? I would like to be able to run the scripts with a `./script.py` instead of calling the whole `python3 script.py`

**Comment by tschmalenbach** [comment]
The problem I see there is needing root privileges to access packages. Installing pip within a conda environment (which is stored in home directory) works fine for me. 

`$ conda create <env_name>` Create env
`$ conda activate <env_name>`
`(env_name)$ conda install pip` Install pip within env
`(env_name)$ ~/anaconda3/envs/env_name/bin/pip install <package>` install packages with pip
`(env_name)$ conda install <package>` install packages with conda

Pip and other packages will need to be reinstalled for every new environment. 

What are the advantages of using `./script.py` instead of `python3 script.py`?

**Comment by manalili18** [comment]
> What are the advantages of using `./script.py` instead of `python3 script.py`?
I have muscle memory for `!.` and not `!p` lol
Other than that, not too much. It behaves nearly identically.
